src/parser/jobs/sch/index.js

```diff
@@ -25,6 +25,11 @@ export const SCHOLAR = new Meta({
 		{user: CONTRIBUTORS.DTHOMASH, role: ROLES.DEVELOPER},
 	],
 	changelog: [
+		{
+			date: new Date('2022-01-25'),
+			Changes: () => <>Update Weaving module text to account for R4 update</>,
```

Comment:
Should this be Broil IV?

Reply:
Er, yes. I'll fix that while I'm running yarn-extract SMN brain rot <_< >_>